students/byungmoon/westargram/users/views.py

```diff
@@ -35,3 +35,18 @@ def post(self, request):
 
         except KeyError:
                 return JsonResponse({"error_message" : "KEY_ERROR"}, status = 400)
+
+class SignInView(View):
+    def post(self,request):
+        try:
+            data     = json.loads(request.body)
+            email    = data['email']
+            password = [data'password']
```

Comment:
틀린 문법이라 테스트가 제대로 되지 않았을 것 같아요! 문법 수정해주세요~
PR 올리시기 전에 체크, 올리신 후에도 한번 더 확인 부탁드립니다!
```suggestion
            password = data['password']
```

Reply:
넵! 확인했습니다